compliance/github/repository_admin_team/README.md

```diff
@@ -0,0 +1,31 @@
+## GitHub.com Repositories without Admin Team
+
+### What it does
+
+This Policy Template gets all the repositories under GitHub.com Organization(s) and creates an incident if any do not have at least 1 Team assigned with `admin` role.  When this requirement is in place, it's easy to identify the responsible owners / team for any given repository.
+
+### Parameters
+1. GitHub.com Organizations to check - Example: `flexera`
+2. Email address to send escalation emails to - Example: `noreply@example.com`
+
+
+### Policy Actions
+
+The following policy actions are taken on any resources found to be out of compliance.
+
+- Send an email report
+
+
+### Required Permissions
+
+This policy requires permissions to access GitHub.com API as the Owner of the Organization(s).  Before applying this policy, create a GitHub.com Personal Access Token under the user with Owner role -- adding the `repo` scopes at minimum, and save the token in the project on Cloud Management as credential named `GITHUB_ORG_ADMIN_ACCESS_TOKEN`.  If you are using other Governance Policies for GitHub.com, you may need to include additional roles to sate the need of all policies which use the same credential.  Optionally, you can generate a token with full permission and avoid any issues.
```

Comment:
"you may need to include additional roles to sate the need of all policies"
Should be "state"